### ISSUE TITLE
Stats: Site owner media uploads seem to be included in Views

### ISSUE THREAD

**Issue by pauljacobson** [Automattic/wp-calypso#48977]
One of our customers contacted us about an unexpected increase in views on their site:

>I have a sudden and ridiculous spike of views on my blog this morning. This is the second time this week that it's happened but this time the number of views is over 400. The concern I have is I've only got one blog post that has gone live so far today with 6 views and somehow someone is viewing the images I've uploaded in bulk with the intent to use in future posts. How is this possible?

#### Steps to reproduce the behavior

1. Head to _My Home_ > _Stats_ in the site's WordPress.com dashboard;
2. Click on the stats for 12 January 2021;
3. Review captured Views for _Posts & Pages_, _Referrers_, and for the _Authors_.
4. See 517 Views for the site owner that aren't accounted for in other Views counts.

#### What I expected to happen

A site owner's visits to the site should not be counted in the site's stats based on [Views and Visitors](https://wordpress.com/support/stats/#views-and-visitors), where we state the following:

> The following are **not** reflected in your stats:
> 
> - Visits from browsers that do not execute javascript or load images
> - GoogleBot and other search engine spiders
> - Visits you make to your own publicly available site (when logged into your account)
> - Visits to a publicly available site by users that are logged in, and listed as members of the site

#### What actually happened

It seems that the user's Media uploads for future posts were included as Views on the site, despite them being the site owner. The user mentioned the following:

>I have been creating reusable blocks with the images I have uploaded but I haven't made any live posts with them yet.

#### Context

6691092-hc and 3642413-zen

### Browser / OS version

The user seems to be using macOS. I will request additional browser and system information.

### Is this specific to the applied theme? Which one?

The user is using Penscratch 2, although this seems to be theme independent.

### Does this happen on simple or atomic sites or both?

This issue has occurred on a WordPress.com Business plan site with plugins.

### Is there any console output or error text?

The only error that I see in Console is the following:

> Firefox can’t establish a connection to the server at wss://public-api.wordpress.com/pinghub/wpcom/me/newest-note-data.

### Level of impact (Does it block purchases? Does it affect more than just one site?)

This is the only report that I'm aware of.

### Screenshot / Video: If applicable, add screenshots to help explain your problem.

This is what I see in Stats:

<img width="1102" alt="Markup 2021-01-17 at 08 30 25" src="https://user-images.githubusercontent.com/2912202/104833228-c8646880-589f-11eb-8008-591a626977f4.png">

**Comment by iamgabrielma** [comment]
They replied with their browser details:
```

Operating systemWindows 10
Javascript enabledYes
Cookies enabledYes
Flash versionNot installed
Java versionNot installed
Websockets supportedYes
WebGL supportedYes
Languageen-US
Browser size1663 x 907
Screen size1680 x 1050
Color depth24 bit
AdBlock enabledNo
Your full user agent string is:
Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.141 Safari/537.36
```

**Comment by chad1008** [comment]
I've gone ahead and closed the ticket related to this: 3642413-zen

Stats have looked normal since the original report. I'll leave the GH issue open in case it warrants further investigation, but unless noted otherwise, we're all set on the Happiness front.

**Comment by cathymcbride** [comment]
Closing since it seems this was an isolated case.
